Spigot-Server-Patches/0319-Add-more-Zombie-API.patch

```diff
@@ -29,24 +29,24 @@ index 12f954723ac2f0058081d810ce83aed99532caad..02a845f99bb445fb4aa66a90273469a8
      public void startDrownedConversion(int i) {
          this.lastTick = MinecraftServer.currentTick; // CraftBukkit
          this.drownedConversionTime = i;
-@@ -226,10 +233,17 @@ public class EntityZombie extends EntityMonster {
+@@ -226,9 +233,16 @@ public class EntityZombie extends EntityMonster {
  
      }
  
 +    public boolean shouldBurnInDay() { return T_(); } // Paper - OBFHELPER
      protected boolean T_() {
-         return true;
-     }
- 
+-        return true;
++        return this.shouldBurnInDay; // Paper - use api value instead
++    }
++
 +    // Paper start
 +    public void setShouldBurnInDay(boolean shouldBurnInDay) {
 +        this.shouldBurnInDay = shouldBurnInDay;
-+    }
+     }
 +    // Paper end
-+
+ 
      @Override
      public boolean damageEntity(DamageSource damagesource, float f) {
-         if (!super.damageEntity(damagesource, f)) {
```

Comment:
You seem to have deleted a line you shouldn't.

Reply:
I just double checked and I definitely did not even touch, let alone delete this line. AFAIK the first sign is the git diff, the second sign (that is missing in the change) would be the patch diff. So lines in patch-commits that only have the first sign should be irrelevant, as they only modified which lines are included in the context of the file in the patch. This probably happened automatically as I only did the following:
```
./paper patch
./paper edit server
<performed the one-line edit>
./paper edit continue
<commited the change>
```